scripts/sanitycheck

```diff
@@ -3208,6 +3211,15 @@ def main():
     global run_individual_tests
     options = parse_arguments()
 
+    if (os.name == 'nt'):
+        print("Running sanitycheck on Windows is not supported yet.")
+        print("https://github.com/zephyrproject-rtos/zephyr/issues/2664")
```

Comment:
If I'm using a supported operation, like --build-only, then I don't want to be warned that what
I'm doing is not supported.

This should either be removed or merged into the below message.